tilequeue/log.py

```diff
@@ -0,0 +1,111 @@
+from enum import Enum
+import json
+import logging
+import sys
+
+
+def make_coord_dict(coord):
+    """helper function to make a dict from a coordinate for logging"""
+    return dict(
+        z=coord.zoom,
+        x=coord.column,
+        y=coord.row,
+    )
+
+
+class LogLevel(Enum):
+    DEBUG = logging.DEBUG
+    INFO = logging.INFO
+    WARNING = logging.WARNING
+    ERROR = logging.ERROR
+
+
+class LogCategory(Enum):
+    PROCESS = 1
+    LIFECYCLE = 2
+    QUEUE_SIZES = 3
+
+
+def log_level_name(log_level):
+    return log_level.name.lower()
+
+
+def log_category_name(log_category):
+    if log_category == LogCategory.QUEUE_SIZES:
+        return 'queue-sizes'
```

Comment:
we seem to use `_` everywhere else, is the `-` deliberate here?

(I just solved a related _ versus - bug in the basemap styles so sensitive to this...)

Reply:
Good call. I checked how we do the tapalcatl logging, and we also prefer `_` there too. Updated in 4310ce1.